README.md

```diff
@@ -103,7 +103,7 @@ Various environment variables are set in the docker-compose file.
 
 1. Start a bash session in the container with:
 
-       docker exec -ti lighthouse_lighthouse_1 bash
+       docker exec -ti lighthouse-lighthouse-1 bash
```

Comment:
Just below the command to execute, using docker ps is suggested. I think, we should keep the version 2 naming. 

"Warning! The names that Docker generates for containers might not be consistent over time. If this doesn't work for you, check the name of the container using docker ps."

Reply:
Maybe just worth adding to the statement below that the naming is based on you using version 2 of Docker Compose if that is the version you observed this under.  Then the person knows how to match their environment to the instructions here instead of feeling the urge to toggle it back to the underscores again.